docs/standard/numerics.md

```diff
@@ -14,63 +14,68 @@ ms.assetid: dfebc18e-acde-4510-9fa7-9a0f4aa3bd11
+|<xref:System.SByte?displayProperty=nameWithType>|Signed|1|-128|127|
+|<xref:System.UInt16?displayProperty=nameWithType>|Unsigned|2|0|65,535|
+|<xref:System.UInt32?displayProperty=nameWithType>|Unsigned|4|0|4,294,967,295|
+|<xref:System.UInt64?displayProperty=nameWithType>|Unsigned|8|0|18,446,744,073,709,551,615|
+
 Each integer type supports a set of standard arithmetic operators. The <xref:System.Math?displayProperty=nameWithType> class provides methods for a broader set of mathematical functions.
 
-You can also work with the individual bits in an integer value by using the <xref:System.BitConverter?displayProperty=nameWithType> class.  
+You can also work with the individual bits in an integer value by using the <xref:System.BitConverter?displayProperty=nameWithType> class.
 
-> [!NOTE]  
+> [!NOTE]
 > The unsigned integer types are not CLS-compliant. For more information, see [Language Independence and Language-Independent Components](language-independence-and-language-independent-components.md).
 
 ## BigInteger
 
 The <xref:System.Numerics.BigInteger?displayProperty=nameWithType> structure is an immutable type that represents an arbitrarily large integer whose value in theory has no upper or lower bounds. The methods of the <xref:System.Numerics.BigInteger> type closely parallel those of the other integral types.
-  
+
 ## Floating-point types
 
-.NET includes three primitive floating-point types, which are listed in the following table:
-  
-|Type|Size (in bytes)|Approximate range|Precision|  
-|----------|--------|---------------------|--------------------|  
-|<xref:System.Single?displayProperty=nameWithType>|4|±1.5 x 10<sup>−45</sup> to ±3.4 x 10<sup>38</sup>|~6-9 digits|  
-|<xref:System.Double?displayProperty=nameWithType>|8|±5.0 × 10<sup>−324</sup> to ±1.7 × 10<sup>308</sup>|~15-17 digits|  
-|<xref:System.Decimal?displayProperty=nameWithType>|16|±1.0 x 10<sup>-28</sup> to ±7.9228 x 10<sup>28</sup>|28-29 digits|  
-  
-Both <xref:System.Single> and <xref:System.Double> types support special values that represent not-a-number and infinity. For example, the <xref:System.Double> type provides the following values: <xref:System.Double.NaN?displayProperty=nameWithType>, <xref:System.Double.NegativeInfinity?displayProperty=nameWithType>, and <xref:System.Double.PositiveInfinity?displayProperty=nameWithType>. You use the <xref:System.Double.IsNaN%2A?displayProperty=nameWithType>, <xref:System.Double.IsInfinity%2A?displayProperty=nameWithType>, <xref:System.Double.IsPositiveInfinity%2A?displayProperty=nameWithType>, and <xref:System.Double.IsNegativeInfinity%2A?displayProperty=nameWithType> methods to test for these special values.
+.NET includes the following primitive floating-point types:
+
+|Type|Size (in bytes)|Approximate range|Precision|Notes|
+|----------|--------|---------------------|--------------------|
+|<xref:System.Half?displayProperty=nameWithType>|2|||Introduced in .NET 5|
```

Comment:
The exact range for `Half` is `+/-65504`.

The number of digits, mirroring what is used in the other columns is `~3-5 digits`. However, that column is very misleading in my opinion and may give users a false sense that any value with 6 significant digits can be exactly represented.

In practice, the lower limit here is that if a string with at most `n` significant digits (`3`, `6`, `15` for `half`, `float`, and `double`, respectively) is converted to a floating-point value and then back to a string with the same number of digits, the strings should match. That is, `float.Parse(s).ToString("G6") == s` should be `true` for any `s` that contains no more than `6` significant digits.
* This isn't guaranteed by the IEEE 754 spec and may depend on the formatting algorithm used

While the upper limit here is that if a floating-point value is converted to a string with that many digits (`5`, `9`, `17` for `half`, `float`, and `double`, respectively), and then converted back to a floating-point value, the result should match the original floating-point value. That is, `float.Parse(x.ToString("G9")).Equals(x)` should be `true` for any `float`.
* This is guaranteed by the IEEE 754 spec

Reply:
I will plan to remove that column then. Thanks!